lib/qs/payload.rb

```diff
@@ -0,0 +1,47 @@
+require 'qs'
+require 'qs/job'
+
+module Qs
+
+  module Payload
+
+    def self.deserialize(serialized_payload)
+      self.deserialize_hash(Qs.deserialize(serialized_payload))
+    end
+
+    def self.deserialize_hash(payload_hash)
+      Qs::Job.new(payload_hash['name'], payload_hash['params'], {
+        :type       => payload_hash['type'],
+        :created_at => Time.at(payload_hash['created_at'].to_i)
+      })
+    end
```

Comment:
@jcredding I don't love that the method name describes what it takes, not what it returns.